core/state/state_object.go

```diff
@@ -79,9 +79,10 @@ type stateObject struct {
 	trie Trie // storage trie, which becomes non-nil on first access
 	code Code // contract bytecode, which gets set when code is loaded
 
-	originStorage Storage // Storage cache of original entries to dedup rewrites
-	dirtyStorage  Storage // Storage entries that need to be flushed to disk
-	fakeStorage   Storage // Fake storage which constructed by caller for debugging purpose.
+	originStorage  Storage // Storage cache of original entries to dedup rewrites
+	pendingStorage Storage // Storage entries that need to be flushed to disk
+	dirtyStorage   Storage // Storage entries that have been modified in the current execution
```

Comment:
For these three, please document their lifetime. 
```
originStorage  Storage // Storage cache of original entries to dedup rewrite. Per transaction
pendingStorage Storage // Storage entries that need to be flushed to disk. Per block
dirtyStorage // Storage entries that have been modified in the current execution. Per transaction
```

Reply:
Done.